src/target/riscv/riscv-013.c

```diff
@@ -2865,10 +2928,10 @@ static int sample_memory_bus_v1(struct target *target,
 			break;
 		}
 
-		size_t sbcs_read_index = riscv_batch_add_dm_read(batch, DM_SBCS);
+		size_t sbcs_read_index = riscv_batch_add_dm_read(batch, DM_SBCS,
+				RISCV_DELAY_SYSBUS_READ);
```

Comment:
The read of SBCS register does not start any bus operation, so I believe the delay should be changed to RISCV_DELAY_BASE.

```suggestion
		size_t sbcs_read_index = riscv_batch_add_dm_read(batch, DM_SBCS,
				RISCV_DELAY_BASE);
```

Reply:
Addressed.